channels-and-providers/sms/clickatell.mdx

```diff
@@ -7,7 +7,7 @@ You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS
 
 # Getting Started
 
-To use the Clickatell provider in the SMS channel, the first step is to create a Clickatell account and add your API key to the Clickatell integration on the Novu platform.
+To use the [Clickatell](https://www.clickatell.com/) provider in the SMS channel, the first step is to [create a Clickatell account](https://www.clickatell.com/sign-up/) and add your API key to the Clickatell integration on the [Novu](https://web.novu.co/integrations) platform.
```

Comment:
'...your API key to the Clickatell integration on the [Novu](https://web.novu.co/integrations) platform.' -> '...on the Novu web dashboard'